### DISCUSSION TITLE
XAML

### DISCUSSION THREAD

**Discussion by sgamesdev** [Leo-Corporation/Gavilya#275] [Q&A] [answered]
how did you do the spacing between game cards on GamesCardsPages.xaml

**Comment by lpeyr** [comment, marked as answer]
I simply added a margin (`Margin="0 10 10 0"`) on the `GameCard` user control:
https://github.com/Leo-Corporation/Gavilya/blob/a9ba67246c0041b84ef70bb42bb7c0adc42d343a/Gavilya/UserControls/GameCard.xaml#L1
